src/django_components/dependencies.py

```diff
@@ -802,27 +826,27 @@ class ComponentDependencyMiddleware:
     """
 
     def __init__(self, get_response: "Callable[[HttpRequest], HttpResponse]") -> None:
-        self.get_response = get_response
+        self._get_response = get_response
```

Comment:
Not related to the bug. Prefixed this method with underscore, so it does not appear in the generated documentation.